pkg/uploader/kopia/snapshot.go

```diff
@@ -152,7 +152,7 @@ func setupPolicy(ctx context.Context, rep repo.RepositoryWriter, sourceInfo snap
 
 // Backup backup specific sourcePath and update progress
 func Backup(ctx context.Context, fsUploader SnapshotUploader, repoWriter repo.RepositoryWriter, sourcePath string, realSource string,
-	forceFull bool, parentSnapshot string, volMode uploader.PersistentVolumeMode, uploaderCfg map[string]string, tags map[string]string, log logrus.FieldLogger) (*uploader.SnapshotInfo, bool, error) {
+	forceFull bool, parentSnapshot string, volMode uploader.PersistentVolumeMode, uploaderCfg map[string]string, tags map[string]string, progress *Progress, log logrus.FieldLogger) (*uploader.SnapshotInfo, bool, error) {
```

Comment:
If this `Backup` function doesn't know Velero's progress, we don't need to force it. We can just call `GetIncrementalSize()` outside this function.